charts/nginx-gateway-fabric/values.yaml

```diff
@@ -104,12 +104,15 @@ terminationGracePeriodSeconds: 30
 ## Tolerations for the NGINX Gateway Fabric pod.
 tolerations: []
 
-##  The nodeSelector of the NGINX Gateway Fabric pod.
+## The nodeSelector of the NGINX Gateway Fabric pod.
 nodeSelector: {}
 
 ## The affinity of the NGINX Gateway Fabric pod.
 affinity: {}
 
+## Whether the platform is openshift or not.
+onOpenshift: false
```

Comment:
Do we want a boolean here, or as @sjberman had suggested prior we do something like `platform`: and we would have values: `kubernetes`, `openshift`, `any future platforms...`

Reply:
Platform seems better to me than a boolean.

Reply:
I agree, having it as a string allows us to expand it in the future if we add any other special cases.